gps_noise_model_folder: '' 
gps_noise_model_multipath: 'gmm_multipath_error_in_function_of_height.pkl'
gps_noise_model_number_of_sat: 'gp_satellites_los_in_function_of_height.joblib'
gps_noise_model_number_of_sat_scaler: 'gp_scaler.joblib'

pseudorange_error_topic: '/gnss_simulator/multipath_error'
number_of_satellite_topic: '/gnss_simulator/number_of_visible_satellites'
ground_truth_topic: '/kingfisher/agiros_pilot/groundtruth/pose'

n_sat_class_0 : 2
n_class: 8

max_time_before_resetting_error: 15.0
max_distance_before_resetting_error: 25.0